.gitignore

```diff
@@ -36,6 +36,7 @@ yarn-error.log
 /libpeerconnection.log
 testem.log
 /typings
+/marketplace-frontend
```

Comment:
is this line necessary to add since there is no more "marketplace-frontend" folder under the root of the repo?

Reply:
Not necessary, just had that there because I accidentally had it in my project.